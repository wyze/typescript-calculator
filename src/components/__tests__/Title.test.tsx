import { h } from 'preact'
import Title from '../Title'
import snapshot from './utils/snapshot'

describe('<Title />', () => {
  it('renders', () => snapshot(<Title />))
})
