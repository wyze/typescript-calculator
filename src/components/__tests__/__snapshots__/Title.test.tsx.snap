// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> renders 1`] = `
<h1
  className="css-tc8z0v"
>
  TypeScript Calculator
</h1>
`;
